runtime/lua/vim/_meta/api_keysets_extra.lua

```diff
@@ -0,0 +1,167 @@
+--- @meta _
```

Comment:
That file is a bit weird... 

I think it might be better to make this a regular commit?

Reply:
Yes, merging did some unexpected changes to this file, I removed the changes, should be fine now! (Also damn you are fast :smile: )